webpack.config.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
       },
       {
         test: /\.s?css$/,
-        exclude: /node_modules/,
+        // We need to ensure sass-loader doesn't try to load flexboxgrid!
+        exclude: [/flexboxgrid/, /node_modules/],
```

Comment:
flexboxgrid was getting pulled in as a node_module before so it was already being excluded. But, now that I've pulled it in and it sits by the FlexGrid stuff, I needed to add it to this clause so sass-loader won't attempt to load it.